config/document_types.yml

```diff
@@ -30,6 +31,7 @@ publication: &publication
   pre_release: true
   path_prefix: /government/publications
   lead_image: false
+  attachments: featured_and_inline
```

Comment:
Doesn't this contradict: https://trello.com/c/dzFScHVI/1498-disable-inline-file-attachments-when-featured-attachments-are-available ?

Reply:
Can you help me understand what you mean here?

Reply:
So it sounds like Publications can only have featured attachments, and not inline ones, so this attribute will need to be changed.

Reply:
Hopefully this makes more sense following our chat: publications can have inline attachments, but the available actions are very different. Using the word `inline` twice does now seem confusing. I've update this to just say `featured`. 

We should be getting design docs on what these two modes mean, and I've had a chat to make sure we're using this terminology in that documentation. The places we use this conditional in the codebase will reflect the design doc, and be another way of defining what it means in executable code. We could have our own, separate doc in the app. I'm worried that would get out-of-date with the others, especially as we've barely started the work to implement the features; so I'd prefer not to add more documentation if possible.